src/components/shared/Entity/Actions/SaveButton.tsx

```diff
@@ -15,14 +15,15 @@ import { FormStatus } from 'stores/FormState/types';
 interface Props {
     closeLogs: Function;
     callFailed: Function;
-    disabled: boolean;
-    taskNames: string[];
     logEvent:
         | CustomEvents.CAPTURE_CREATE
+        | CustomEvents.COLLECTION_CREATE
         | CustomEvents.MATERIALIZATION_CREATE
         | CustomEvents.CAPTURE_EDIT
         | CustomEvents.MATERIALIZATION_EDIT;
+    disabled?: boolean;
```

Comment:
The `disabled` property was made optional because logic has not been defined to conditionally set it and I did not want to pass a fixed boolean _initially_. In retrospect, it does not make sense to make this property optional given it will likely be conditionally set in the near future.